.github/actions/setup/action.yml

```diff
@@ -33,10 +33,17 @@ runs:
       - name: Fetch base ref to find merge-base for GIB
         shell: bash
         run: .github/bin/git-fetch-base-ref.sh
-      - name: Free additional disk space
+      - name: Maximize build space
         if: ${{ format('{0}', inputs.cleanup-node) == 'true' }}
-        shell: bash
-        run: ./.github/bin/free-disk-space.sh
+        uses: easimon/maximize-build-space@c28619d8999a147d5e09c1199f84ff6af6ad5794
```

Comment:
since you raised this PR from within the repo, should i assume the trust has already been built?